index.js

```diff
@@ -211,42 +221,75 @@
 
   /**
    * Game Constructor
-   * @param {number} speed 
+   * @param {number} fps frames per second 
+   * @param {number} gameSpeed
+   * @param {number} computerPaddleSpeed how fast the computers paddle can react
    */
-  function Game(speed) {
-    this.ball = new Ball();
-    this.playerPaddle = new Paddle("left");
-    this.computerPaddle = new Paddle("right", computerPaddleSpeed);
-    this.framesPerSecond = speed;
-  };
+  class Game {
+    constructor(fps, computerPaddleSpeed) {
```

Comment:
If you were to make the other classes take configuration options, then you would also want to be able to pass those options in to the game constructor which would then pass them down in to the object constructors during its setup here.
```suggestion
    constructor(fps, computerPaddleSpeed, opts) {
```